File: Java Programming/Single_Inheritance.java
class Single_Inheritance 
{

    void get_values(int a, int b)
    {

        int c=0;
        c=a+b;
        System.out.println("the addition is "+c);

    }
};
class display extends Single_Inheritance
{
    public static void main(String [] args)
    {
        display d = new display();
        d.get_values(10,20);
    }
}

File: CPP Programming/Single_Inheritance.cpp
#include<iostream>
using namespace std;
int a, b, c;
class Values
{
    public:
    void get_values()
    {
        cout<<"Enter the value of a"<<endl;
        cin>>a;
        cout<<"Enter the value of b"<<endl;
        cin>>b;
    }

};
class Operation : public Values
{
    public:
    void operation()
    {
        c=a+b;
        cout<<"The Addition is "<<c<<endl;
    }
};
int main()
{
    Operation op;
    op.get_values();
    op.operation();
}